configure.ac

```diff
@@ -51,4 +51,22 @@ PKG_CHECK_MODULES([libbase58],[libbase58])
 
 AC_CHECK_LIB([ws2_32], [strchr])
 
+#libgcrypt necessary to build example.c with "make example"
+#check for libgcrypt:
+m4_ifdef([AM_PATH_LIBGCRYPT], [
+	AM_PATH_LIBGCRYPT([],
+		[
+		AC_MSG_RESULT([\
+			libgcrypt available; compile example.c with "make example"])
+			[can_compile_example=yes]
+		],[
+		AC_MSG_RESULT([\
```

Comment:
These should be AC_MSG_NOTICE/AC_MSG_WARN, but IMO it's best not to print anything extra at all. Maybe throw a dummy example rule in Makefile.am that prints an explanation why it isn't a valid target?

```
example:
    @echo "libgcrypt is required to build the example, but was not found"
.PHONY: example
```